test/sanity/language/propertiesTestCase/propertiesInClasses.wtest

```diff
@@ -9,7 +9,7 @@ class Ave {
 
   method volar() {
     energia -= 10
-    vecesQueVolo++
+    vecesQueVolo += 1
```

Comment:
good catch! I saw the issue regarding `++` & `--`  and fortunately you didn't implement it in wollok-ts